.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+  - package-ecosystem: cargo
+    directory: /
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: gomod
+    directory: /
+    schedule:
+      interval: weekly
```

Comment:
The Go implementation has zero dependencies

Reply:
BELT AND SUSPENDERS BABEEEE

Reply:
Happy to remove?

Reply:
No, you can keep it!